rocky/reports/templates/report_overview/report_history_table.html

```diff
@@ -2,141 +2,148 @@
+                                            <li class="label tags-color-{{ report_type|get_report_type_label_style }}">{{ report_type|get_report_type_name }}</li>
+                                        {% endif %}
+                                        {% if forloop.counter0 == 2 %}
+                                            <li class="label tags-color-grey-2">+ {{ report.report_type_summary|length|add:"-2" }}</li>
+                                        {% endif %}
                                     {% endfor %}
-                                </tbody>
-                            </table>
-                            <div class="button-container">
-                                {% if report.total_children_reports > 5 %}
-                                    <a href="{% url "subreports" organization.code %}?report_id={{ report.parent_report.reference }}"
-                                       class="button">{% translate "View all subreports" %}</a>
+                                {% else %}
+                                    <li class="label tags-color-{{ report.parent_report.report_type|get_report_type_label_style }}">
+                                        {{ report.parent_report.report_type|get_report_type_name }}
+                                    </li>
                                 {% endif %}
-                            </div>
+                            </ul>
+                        </td>
+                        <td>
+                            {% if report.parent_report.input_oois|length == 1 %}
+                                <a href="{% ooi_url "ooi_detail" report.parent_report.input_oois.0 organization.code query=ooi.mandatory_fields %}">{{ report.parent_report.input_oois.0|human_readable }}</a>
+                            {% elif report.total_objects == 1 %}
+                                <a href="{% ooi_url 'ooi_detail' report.children_reports.0.input_oois.0 organization.code query=ooi.mandatory_fields %}">{{ report.children_reports.0.input_oois.0|human_readable }}</a>
+                            {% else %}
+                                {{ report.total_objects }}
+                            {% endif %}
+                        </td>
+                        <td class="nowrap">{{ report.parent_report.observed_at|date }}</td>
+                        <td class="nowrap">{{ report.parent_report.date_generated }}</td>
+                        <td>
+                            {% if report.children_reports %}
+                                <button type="button"
+                                        class="expando-button icon ti-chevron-down"
+                                        data-icon-open-class="icon ti-chevron-down"
+                                        data-icon-close-class="icon ti-chevron-up"
+                                        data-close-label="{% translate "Close children report object details" %}"
+                                        aria-expanded="false"></button>
```

Comment:
The open label is missing here:

```suggestion
                                        aria-expanded="false">
                                        {% translate "Open children report object details" %}
                                </button>
```